src/convertToObject.js

```diff
@@ -6,7 +6,26 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const convertedObj = {};
```

Comment:
[CODE STYLE] Avoid variable namings like `convertedObj`. It will decrease code readability. Use more specific names. For example, `cssStylesObject`.